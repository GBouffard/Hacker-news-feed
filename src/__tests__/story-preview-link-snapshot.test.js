import React from 'react';
import StoryPreviewLink from '../components/story-preview-link';
import renderer from 'react-test-renderer';

describe('StoryPreviewLink ', () => {
  const component = renderer.create(
    <StoryPreviewLink
      by='Guillaume'
      id={123456789}
      score={99}
      title='this is a snapshot test' />
  );
  let tree = component.toJSON();

  it('renders correctly', () => {
    expect(tree).toMatchSnapshot();
  });
});