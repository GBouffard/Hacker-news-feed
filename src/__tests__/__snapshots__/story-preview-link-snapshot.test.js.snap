// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StoryPreviewLink  renders correctly 1`] = `
<a
  className="story-preview__link"
  href="#popup-123456789"
>
  <div
    className="story-preview__score"
  >
    99
  </div>
  <div
    className="story-preview__title-and-author-container"
  >
    <div
      className="story-preview__title"
    >
      this is a snapshot test
    </div>
    <div
      className="story-preview__author"
    >
      by: 
      Guillaume
    </div>
  </div>
</a>
`;
